### DISCUSSION TITLE
Avoidance criteria

### DISCUSSION THREAD

**Discussion by Leeb64** [devemux86/cruiser#143] [Q&A] [answered]
Routing:
The selectable avoidance criteria have no effect on most routing services!
It works with Valhalla and offline Brouter.
It would be helpful if only the selectable avoidance criteria would be displayed.

**Comment by devemux86** [comment]
@Leeb64 thanks for the suggestion!

Avoidance options currently work with the routing services:

- BRouter (offline)
- MapQuest
- OpenRouteService
- Valhalla

Indeed, we could enable these settings when the relevant service is selected.

**Reply by 0709wiwiwi** [reply]
Another great example that you first and foremost select the desired Service and then the appropriate profiles and the options.

**Comment by devemux86** [comment, marked as answer]
Now each routing service has separate settings and avoidances.
